src/wms/src/main/java/org/geoserver/wms/legendgraphic/LegendMerger.java

```diff
@@ -299,7 +300,7 @@ public static BufferedImage mergeLegends(Rule[] rules, GetLegendGraphicRequest r
         }
 
         if (layout == LegendLayout.VERTICAL) {
-            Column[] columns = createColumns(nodes, mergeOptions.getColumnHeight(), mergeOptions.getColumns());
+            Column[] columns = createNRColumns(nodes, mergeOptions.getColumnHeight(), mergeOptions.getColumns(), req);

```

Comment:
To better match the pre-existing naming convention, this should probably be called createColumns, and the other method should be called createRasterColumns (since we are in the mergeLegends method, and the other one is called from mergeRasterLegends)